packages/aws-cdk/lib/api/provisioner.ts

```diff
@@ -0,0 +1,44 @@
+import cxapi = require('@aws-cdk/cx-api');
+import { debug } from '../logging';
+import { deserializeStructure } from '../serialize';
+import { Mode } from './aws-auth/credentials';
+import { SDK } from './util/sdk';
+
+export type Template = { [key: string]: any };
+
+/**
+ * Interface for provisioners
+ *
+ * Provisioners apply templates to the cloud infrastructure.
```

Comment:
Currently this is more of an inspection than an application...

Reply:
It's aspirational and a guide to future maintainers.